backend/trafficrules/src/main/resources/db/migration/V2__Alter_table_users.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `users`
+	ADD COLUMN `registered` DATETIME(6) NULL DEFAULT NULL AFTER `name`,
+	ADD COLUMN `last_activity` DATETIME(6) NULL DEFAULT NULL AFTER `registered`;
```

Comment:
Should these fields be nullable and with null default value? Maybe it's better to make it no nullable with the current time as a default?